### DISCUSSION TITLE
Could not scan for classes inside "...polyfill-php73/Resources/stubs" which does not appear to be a fil e nor a folder

### DISCUSSION THREAD

**Discussion by Leshamer** [symfony/symfony#53847] [Q&A]
Hi all, I am new to working with composer and recently encountered this error in a project.
![image](https://github.com/symfony/symfony/assets/99595862/1a6a14f8-6a63-4dc6-9284-edf2e2985d75)

 I tried deleting vendor folder and composer.lock, updating composer version, downgrading composer version to 2.0.4, I cleaned composer cache. But every time I get the same error.

**Comment by kAlvaro** [comment]
What's your operating system and command-line PHP version? Are the files from "vendor/symfony/polyfill-php80/Resources/stubs" actually missing if you look at them in your file manager? Have you tried running composer from a regular terminal rather than inside your IDE?

If you can share your composer.json file (you can strip or redact project details like `description`, `authors`...) we can try to reproduce it. But I feel that these kind of problems are typically created by overzealous security tools (antivirus, malware scanners, firewalls...).

**Comment by MatTheCat** [comment]
What's your composer version? https://github.com/symfony/polyfill/issues/174 sounds similar but it has been fixed in composer 2.0.6 (which is more than 3 years old).
